packages/inventory/src/utils/build-query.ts

```diff
@@ -69,6 +69,10 @@ export function buildQuery<TWhereKeys extends object, TEntity = unknown>(
     query.order = buildOrder(config.order) as FindOptionsOrder<TEntity>
   }
 
+  if (config.withDeleted) {
```

Comment:
todo: you can move that line 44 `|| config.withDeleted`, any reason why the inventory has this util and do not use the one in medusa utils?

Reply:
The one in @medusajs/utils doesn't seem to do the same thing as this one.

Reply:
Could you verify, cause it looks like it does the same

Reply:
I ll get another look later, for some reason i had in mind that this specific function was also part of the utils. Discard for now